wrappers/python/simtk/openmm/app/forcefield.py

```diff
@@ -2439,7 +2439,7 @@ def createForce(self, sys, data, nonbondedMethod, nonbondedCutoff, args):
 
         for atom in data.atoms:
             self.force.addParticle((typeToMergedType[data.atomType[atom]],))
-        self.force.setUseLongRangeCorrection(True)
+        self.force.setUseLongRangeCorrection(bool(args.get('useLongRangeCorrection', False)))
```

Comment:
This change is incorrect.  On any force field other than CHARMM, the correction should be enabled by default.

Reply:
OK, let me fix that.

Reply:
A better solution would be to make this an attribute of the `<LennardJonesForce>` tag.  It could be processed similarly to the existing `lj14scale` attribute: see https://github.com/openmm/openmm/blob/master/wrappers/python/simtk/openmm/app/forcefield.py#L2359-L2366.

Then we would need to add that attribute to all the CHARMM36 files.  Those files were generated with ParmEd.  For the moment it would probably be reasonable to just add the attribute by hand, but ParmEd would also need to get updated to produce correct files in the future.